adapters/Vungle/VungleAdapter/GADMAdapterVungleRouter.m

```diff
@@ -456,6 +458,15 @@ - (void)initialized:(BOOL)isSuccess error:(nullable NSError *)error {
   [_initializingDelegates removeAllObjects];
 }
 
+- (void)checkSamePlacementIDwithNilUniquePubRequestID:(nonnull GADMAdapterVungleBannerRequest *)vungleBannerRequest {
```

Comment:
just confirming - this method is just providing log output to publisher explaining that they should use Vungle network extras to support banner refresh?

Reply:
@crbinsf Hi Clarke, yes, actually this log is requested by AdMob. And Android side also added the same log to ask publishers that they should use Vungle network extras to support banner refresh. Thanks a lot. :)